snippets/base/models.py

```diff
@@ -851,12 +851,23 @@ class SimpleTemplate(Template):
         on_delete=models.SET_NULL,
         verbose_name='Section Title Icon',
         related_name='simple_section_icons',
-        help_text=('Section title icon. 32x32px. PNG. '
+        help_text=('Section title icon. 16x16px. PNG. '
+                   'section_title_text must also be specified to display.'),
+    )
+    section_title_icon_dark_theme = models.ForeignKey(
```

Comment:
We won't be using dark icons anytime soon according to the snippets editors, so I suggest that we don't include these options to keep it simple(r)